tdrs-backend/tdpservice/email/templates/upcoming-submission-deadline.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html' %}
+{% block content %}
+
+<!-- Body copy -->
+<p>Hello {{ first_name }},</p>
+<p>This is a friendly reminder that your data files are due in 5 days. </p>
+<p>Please sign in to the <a href="https://tanfdata.acf.hhs.gov/" target="_blank">TANF Data Portal</a> 
+to upload and submit data files for <b>Fiscal Year {{ fiscal_year }} - {{ fiscal_quarter}}</b> by <b>{{ submission_deadline }}.</b></p>
+
+<!-- This link uses descriptive text to inform the user what will happen with the link is tapped. -->
+<!-- It also uses inline styles since some email clients won't render embedded styles from the head. -->
+<!-- <a href="" style="color: #336a90; text-decoration: underline;">Reset your password now</a> -->
+<p><a class="button" href="https://tdp-frontend-a11y.app.cloud.gov/" style="background-color:#336a90;border-radius:4px;color:#ffffff;display:inline-block;font-family:sans-serif;font-size:18px;font-weight:bold;line-height:60px;text-align:center;text-decoration:none;width: auto; padding-left: 24px; padding-right: 24px;-webkit-text-size-adjust:none;">Submit your data files</a></p>
+
+<b>Need help?</b><br>
+We're here for you! Check out the <a href="http://tdp-project-updates.app.cloud.gov/knowledge-center/" target="_blank">TDP Knowledge Center</a> for specific gudiance on  <a href="https://tdp-project-updates.app.cloud.gov/knowledge-center/uploading-data.html" target="_blank">Submitting Data Files</a> and <a href="https://tdp-project-updates.app.cloud.gov/knowledge-center/faq.html">Frequently Asked Questions</a>.
+<p> 
+TDP is now the only method for data submissions; you should not be submitting data through SFTP or Cyberfusion. Reach out to the <a href="mailto:TANFData@acf.hhs.gov" target="_blank"> TDP support team</a> by replying to this email and we will follow up with you directly.</p>
```

Comment:
@jtimpe @reitermb they will not be able to reply to this email since it's coming from no-reply. Could drop the part after "by replying..."

Reply:
i made an attempt to update the language as you described, @reitermb please review :D

Reply:
done!